File: 1633-minimum-number-of-increments-on-subarrays-to-form-a-target-array/minimum-number-of-increments-on-subarrays-to-form-a-target-array.cs
public class Solution {
    public int MinNumberOperations(int[] target) {
        int result=target[0];
        for(int i=1; i<target.Length; i++){
            if(target[i]>target[i-1]){
                result+=target[i]-target[i-1];
            }
        }
        return result;
    }
}

File: 1633-minimum-number-of-increments-on-subarrays-to-form-a-target-array/minimum-number-of-increments-on-subarrays-to-form-a-target-array.py
class Solution(object):
    def minNumberOperations(self, target):
        """
        :type target: List[int]
        :rtype: int
        """
        r=target[0]
        for i in range(1,len(target)):
            if target[i]>target[i-1]:
                r+=target[i]-target[i-1]
        return r